### DISCUSSION TITLE
Unaware of git lfs

### DISCUSSION THREAD

**Discussion by CodeShaper13** [community/community#22172] [New to GitHub]
<p>I am a beginner to git so if I’m using the wrong terms feel free to point it out.</p>
<p>Over a year ago I created a github repo for a large Unity project using a plugin <a href="https://unity.github.com/" rel="noopener nofollow ugc">https://unity.github.com/</a> and was unaware that LFS was even a thing.  I had a computer issue unrelated to git and my project on my machine was deleted.  Today I wanted to get my project back and discovered that I can’t because “this repository is over its data quota” which is confusing me because I haven’t touched this in nearly a year and I don’t remember agreeing or setting anything up involving LFS (I didn’t even know it was a thing until today).</p>
<p>How do I get MY project back without paying for something I didn’t even know existed?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>From the looks of things, you’re using LFS in several different repositories - many of which are private. Depending on whether or not those repositories are important to you, the solutions will vary. Deleting larger repositories you no longer use could be a quick way to see if maybe something you don’t care about anymore is causing you some trouble.</p>
<p>If you don’t want to do that, I’d recommend going to private support, where your account and its contents can be discussed directly (and not in public). They’ll be able to suggest the best way forward to getting your account back under the LFS limit.</p>
<p>Please <a href="https://support.github.com/contact">open a ticket</a>. The way you have described it here is great, so you don’t need to worry about terminology. You’ll be understood.</p>

**Comment by CodeShaper13** [comment]
<p>Thanks for the quick response.  How do I see if a repo is using LFS?  I looked through the settings and didn’t find anything.</p>

**Comment by canuckjacq** [comment]
<p>I’m not sure that’s something you can find easily - if you open a ticket you’ll be able to get a list of all the repositories using LFS.</p>

**Comment by CodeShaper13** [comment]
<p>I opened a ticket and got everything solved.  Thank you for your help.</p>
